docs/multidomain-site-example/site.conf

```diff
@@ -8,12 +8,14 @@
   regdom = 'DE',
 
   wifi24 = {
+    beacon_interval = 1000,
     mesh = {
       mcast_rate = 12000,
     },
   },
 
   wifi5 = {
+    beacon_interval = 500,
```

Comment:
- As long as we don't have any experience with this setting, it should not appear in our example configs, or only as a comment
- If we leave this as a comment, the non-multidomain example should be adjusted as well.

Reply:
Removed this setting